server/memphis_handlers_user_mgmt.go

```diff
@@ -402,6 +434,131 @@ func (umh UserMgmtHandler) AuthenticateNatsUser(c *gin.Context) {
 	c.IndentedJSON(200, gin.H{})
 }
 
+func (umh UserMgmtHandler) GetSignUpFlag(c *gin.Context) {
+	exist, err := isOnlyRootUserExist()
+	if err != nil {
+		serv.Errorf("GetSignUpFlag error: " + err.Error())
+		c.AbortWithStatusJSON(500, gin.H{"message": "Server error"})
+		return
+	}
+	var message string
+	if exist {
+		message = "Only root user exists"
+	} else {
+		message = "More than root user exists"
+	}
+	serv.Warnf(message)
+	c.IndentedJSON(200, gin.H{"exist": exist, "message": message})
+}
+
+func (umh UserMgmtHandler) AddUserSignUp(c *gin.Context) {
+	exist, err := isOnlyRootUserExist()
+	if err != nil {
+		serv.Errorf("AddUserSignUp error: " + err.Error())
+		c.AbortWithStatusJSON(500, gin.H{"message": "Server error"})
+		return
+	}
+	if !exist {
+		serv.Warnf("More than root user exists")
+		c.IndentedJSON(401, gin.H{"message": "More than root user exists"})
+		return
+	} else {
+		serv.Warnf("Only root user exists")
+		var body models.AddUserSchema
+		ok := utils.Validate(c, &body, false, nil)
```

Comment:
schema should be validated first thing in this function, as happens in other handlers